pkg/api/v1/testkube/model_test_workflow_extended.go

```diff
@@ -80,3 +80,19 @@ func (w *TestWorkflow) GetObjectRef() *ObjectRef {
 
 func (w *TestWorkflow) QuoteWorkflowTextFields() {
 }
+
+func (w TestWorkflow) GetName() string {
```

Comment:
For RBAC we need Workflow, Webhook, Trigger and WorkflowTemplate. The latter seems to be missing from this PR. If we'd add it then we have to avoid making another release just for that.